settings.gradle

```diff
@@ -0,0 +1,26 @@
+pluginManagement {
+    repositories {
+        google {
+            content {
+                includeGroupByRegex("com\\.android.*")
+                includeGroupByRegex("com\\.google.*")
+                includeGroupByRegex("androidx.*")
+            }
+        }
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+dependencyResolutionManagement {
+    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+            url "https://artifacts.sency.ai/artifactory/debug"
```

Comment:
@SencyCaspit debug?

Reply:
@ofergoldstein yes, we still testing rc0.1.7.0, after approving the demo app and general implementation of smkit i could publish to release. that sounds good?